src/routes/MainRoute/MainRouteStyled.js

```diff
@@ -0,0 +1,141 @@
+import styled from 'styled-components';
+import SmallLogo from '@/images/small_logo.svg';
+import Miniature from '@/images/main_page_miniature.svg'
+import { Typography, Box } from '@mui/material';
+
+export const MainPageLayout = styled(Box)`
+box-sizing: border-box;
```

Comment:
Wrong indents. Seems like prettier plugin doesn't work for you.

Reply:
Indents fixed. Something really wrong with prettier.